ddl/ddl_db_test.go

```diff
@@ -995,7 +1005,7 @@ LOOP:
 	c.Assert(count, Greater, int64(0))
 }
 
-// This test is for insert value with a to-be-dropped column when do drop column.
+// testDropColumn2 This test is for insert value with a to-be-dropped column when do drop column.
```

Comment:
s/testDropColumn2 This test is for insert value/testDropColumn2 is for inserting value

Reply:
@yangwenmai friendly ping